tools/validate-xep0001-conformance.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+# Checks if provided file (filename to be provided as first argument) conforms to XEP-0001.
+# See https://github.com/xsf/xeps/issues/1235
+#
+# Expected to be executed from the directory that holds all xep XML files.
+# Requires one argument: the file name of the xep to be validated, eg:
+#
+# $ tools/validate-xep0001-conformance.sh xep-0010.xml
+#
+# exit status will be non-zero upon validation failure.
+#
+# requires: bash, xmllint
+
+set -e
```

Comment:
Please change this to `set -euo pipefail` and fix any resulting issues. `-u` shouts at you if you use an undefined variable (very useful to catch typos) and `-o pipefail` improves error handling by also failing if any of the commands in a pipe fail.